### PR TITLE
"Fixed local wait issue and added continue shopping test on checkout"

### PR DESCRIPTION
**Pull request by KashyapDave95**
"Fixed local wait issue and added continue shopping test on checkout"

### CODE REVIEW

**Review by SohamQAKrish — CHANGES_REQUESTED** [review]
Do not push all the details. Remove unnecessary files from here.

File: src/main/java/common/UtilitiesCommon.java [hunk 1]
```diff
@@ -184,6 +184,7 @@ public static void setupLogger() {
         logger = LogManager.getLogger(UtilitiesCommon.class);
 	}
 
+
```
[the remark this answers is not in the document]
Comment (KashyapDave95) [reply src/main/java/common/UtilitiesCommon.java]: Removed space.

File: src/main/java/page/bateel/BateelGuestCheckoutPage.java [hunk 2]
```diff
@@ -93,4 +93,9 @@ public static void clickPlaceOrderButton() {
         UtilitiesCommon.click(BateelCheckoutGuestPageEnum.BATEEL_GUEST_PLACEORDER_ENUM_XPATH);
 
     }
+    public static void clickOnContinueShopButton() {
+
```
[the remark this answers is not in the document]
Comment (KashyapDave95) [reply src/main/java/page/bateel/BateelGuestCheckoutPage.java]: Removed spaces.

**Comment by SohamQAKrish** [comment]
Closing this PR as we can not commit 70 files. Re-raise the PR after making the changes.

@KashyapDave95  @rammohan-krish-QA

**Pull request closed without merging**